### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by AlexGrytsai**

### CODE REVIEW

**Review by NikOneZ1 — CHANGES_REQUESTED** [review]
Why is there a pull request in the draft?

**Comment by AlexGrytsai** [comment]
> Why is there a pull request in the draft?

Fixed

File: app/main.py [hunk 1]
```diff
@@ -1 +1,46 @@
...
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        if self.is_hungry:
+            print(f"Eating {self.appetite} food points...")
+            self.is_hungry = False
+            return self.appetite
+        else:
+            return 0
```
Comment (EdAlekseiev) [inline app/main.py:20]:
```suggestion
        if self.is_hungry:
            print(f"Eating {self.appetite} food points...")
            self.is_hungry = False
            return self.appetite
        return 0
```

File: app/main.py [hunk 2]
```diff
@@ -1 +1,46 @@
...
+    def bring_slippers() -> None:
+        print("The slippers delivered!")
+
+
+def feed_animals(animals: list) -> int:
+    sum_food_points = 0
+    for animal in animals:
+        if animal.is_hungry:
+            sum_food_points += animal.feed()
+    return sum_food_points
```
Comment (EdAlekseiev) [inline app/main.py:46]: Rewrite this method to one line with using function `sum`

**Review by EdAlekseiev — CHANGES_REQUESTED** [review]

**Review by EdAlekseiev — APPROVED** [review]
Good job

**Pull request closed without merging**